zeebe/snapshot/src/main/java/io/camunda/zeebe/snapshots/impl/SfvChecksumImpl.java

```diff
@@ -170,6 +172,24 @@ public void updateFromChecksum(final Path filePath, final long checksum) {
     checksums.put(fileName, checksum);
   }
 
+  @Override
+  public void updateFromSnapshotChunk(final SnapshotChunk chunk) {
+    final String fileName = chunk.getChunkName();
+
+    if (chunk.getFileBlockPosition() == 0) {
+      combinedChecksum.update(fileName.getBytes(UTF_8));
+    }
+
+    partialChecksums.putIfAbsent(fileName, new CRC32C());
+    partialChecksums.get(fileName).update(chunk.getContent());
+    combinedChecksum.update(chunk.getContent());
+
+    if (chunk.isLastFileBlock()) {
+      checksums.put(fileName, partialChecksums.get(fileName).getValue());
```

Comment:
:upside_down_face: You can inline the `partialChecksums.remove(fileName)` here because it returns the value.